README.md

```diff
@@ -1,6 +1,6 @@
 # CORE Network Simulator
 
-This image is an inspirated on the work of [Stuard Mardsen](https://github.com/stuartmarsden/dockerCoreEmu) CORE EMU image and his recommendations on [Eriberto packages](http://eriberto.pro.br/core/
+This image is an inspirated on the work of [Stuart Mardsen](https://github.com/stuartmarsden/dockerCoreEmu) CORE EMU image and his recommendations on [Eriberto packages](http://eriberto.pro.br/core/
```

Comment:
No problem. Since I was updating the readme anyway thought I would just change it.